### ISSUE TITLE
v1.95.1 Mount Volume Error. Cannot start. 

### ISSUE THREAD

**Issue by joe-cole1** [immich-app/immich#7303]
### The bug

After upgrading to v1.95.1, I have completed step 3 in breaking changes, then taken down and up the stack for step 4. 

However, it does not start correctly and it's due to my mounted library for some reason. It never gets to the re-index log and the server does not start successfully. 

The following log just repeats forever. 

```
[Nest] 7  - 02/21/2024, 11:43:27 AM     LOG [CommunicationRepository] Initialized websocket server
[Nest] 7  - 02/21/2024, 11:43:27 AM     LOG [SystemConfigService] LogLevel=log (set via system config)
[Nest] 7  - 02/21/2024, 11:43:28 AM     LOG [LibraryService] Starting to watch library cc9eca5c-7538-498f-b803-8919df882af3 with import path(s) /mnt/media/joe-cell-old
node:internal/errors:563
    ErrorCaptureStackTrace(err);
    ^
Error: ENOSPC: System limit for number of file watchers reached, watch '/mnt/media/joe-cell-old'
    at FSWatcher.<computed> (node:internal/fs/watchers:247:19)
    at Object.watch (node:fs:2473:36)
    at createFsWatchInstance (/usr/src/app/node_modules/chokidar/lib/nodefs-handler.js:119:15)
    at setFsWatchListener (/usr/src/app/node_modules/chokidar/lib/nodefs-handler.js:166:15)
    at NodeFsHandler._watchWithNodeFs (/usr/src/app/node_modules/chokidar/lib/nodefs-handler.js:331:14)
    at NodeFsHandler._handleDir (/usr/src/app/node_modules/chokidar/lib/nodefs-handler.js:567:19)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async NodeFsHandler._addToNodeFs (/usr/src/app/node_modules/chokidar/lib/nodefs-handler.js:617:16)
    at async /usr/src/app/node_modules/chokidar/index.js:451:21
    at async Promise.all (index 0)
Emitted 'error' event on FSWatcher instance at:
    at FSWatcher._handleError (/usr/src/app/node_modules/chokidar/index.js:647:10)
    at NodeFsHandler._addToNodeFs (/usr/src/app/node_modules/chokidar/lib/nodefs-handler.js:645:18)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async /usr/src/app/node_modules/chokidar/index.js:451:21
    at async Promise.all (index 0) {
  errno: -28,
  syscall: 'watch',
  code: 'ENOSPC',
  path: '/mnt/media/joe-cell-old',
  filename: '/mnt/media/joe-cell-old'
}
Node.js v20.11.0
```

### The OS that Immich Server is running on

Synology DSM 7.3

### Version of Immich Server

v1.95.1

### Version of Immich Mobile App

v1.95.1

### Platform with the issue

- [X] Server
- [ ] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
version: "3.8"

name: immich

services:
  immich-server:
    container_name: immich_server
    image: ghcr.io/immich-app/immich-server:${IMMICH_VERSION:-release}
    command: ["start.sh", "immich"]
    volumes:
      - ${UPLOAD_LOCATION}:/usr/src/app/upload
      - /etc/localtime:/etc/localtime:ro
      - /volume1/photo/MobileBackup/joe:/mnt/media/joe-cell-old:ro
      - /volume1/photo/Professional:/mnt/media/professional:ro
    env_file:
      - stack.env
    ports:
      - 2283:3001
    depends_on:
      - redis
      - database
    restart: always
    
  immich-microservices:
    container_name: immich_microservices
    image: ghcr.io/immich-app/immich-server:${IMMICH_VERSION:-release}
    devices:
      - /dev/dri/renderD128:/dev/dri/renderD128
      - /dev/dri/card0:/dev/dri/card0
    command: ["start.sh", "microservices"]
    volumes:
      - ${UPLOAD_LOCATION}:/usr/src/app/upload
      - /etc/localtime:/etc/localtime:ro
      - /volume1/photo/MobileBackup/joe:/mnt/media/joe-cell-old:ro
      - /volume1/photo/Professional:/mnt/media/professional:ro
    env_file:
      - stack.env
    depends_on:
      - redis
      - database
    restart: always

  immich-machine-learning:
    container_name: immich_machine_learning
    image: ghcr.io/immich-app/immich-machine-learning:${IMMICH_VERSION:-release}
    volumes:
      - model-cache:/cache
    env_file:
      - stack.env
    restart: always

  redis:
    container_name: immich_redis
    image: redis:6.2-alpine@sha256:afb290a0a0d0b2bd7537b62ebff1eb84d045c757c1c31ca2ca48c79536c0de82
    restart: always

  database:
    container_name: immich_postgres
    #image: tensorchord/pgvecto-rs:pg14-v0.1.11@sha256:0335a1a22f8c5dd1b697f14f079934f5152eaaa216c09b61e293be285491f8ee 
    image: tensorchord/pgvecto-rs:pg14-v0.2.0@sha256:90724186f0a3517cf6914295b5ab410db9ce23190a2d9d0b9dd6463e3fa298f0
    env_file:
      - stack.env
    environment:
      POSTGRES_PASSWORD: ${DB_PASSWORD}
      POSTGRES_USER: ${DB_USERNAME}
      POSTGRES_DB: ${DB_DATABASE_NAME}
    volumes:
      - pgdata:/var/lib/postgresql/data
    restart: always
    
volumes:
  pgdata:
  model-cache:
```

### Your .env content

```Shell
UPLOAD_LOCATION=/volume1/photo/Immich
IMMICH_VERSION=release
TYPESENSE_API_KEY=
DB_PASSWORD=
DB_HOSTNAME=
DB_USERNAME=
DB_DATABASE_NAME=immich
REDIS_HOSTNAME=immich_redis
```

### Reproduction steps

```bash
Step 4: Terminate and restart the stack
Bring the stack down (or terminate with ctrl + c) with:

docker compose down
Then bring it back up:

docker compose up
You'll run into a message saying:

[DatabaseRepository] Could not reindex index face_index. Attempting to auto-fix.
image.png (view on web)

This is normal. The server will do some magic and start to work.
```

**Comment by joe-cole1** [comment]
This has not been an issue for many months. What changed?

**Comment by joe-cole1** [comment]
For those who run into the same problem, here is how I solved it on a Synology DS920+ running DSM 7.2.1

I created a script in Control Panel -> Task Scheduler that runs at boot. It runs under the user root with the command:

```
sh -c '(sleep 90 && echo 500000 > /proc/sys/fs/inotify/max_user_watches)&'
```

the "500000" is configurable. Not sure what the default is on Synology, but this worked fine for me after running the script manually and then restarting my Immich container.
